products/knownprojects/python/download.py

```diff
@@ -14,13 +14,13 @@
 def download_s3_source_file(file_name: str) -> None:
     s3_filepath = f"{S3_SOURCE_HOUSING_TEAM_DIR}/raw/{file_name}"
     save_file_path = f"{RAW_DATA_PATH}/{file_name}"
-    client().download_file(S3_SOURCE_BUCKET, s3_filepath, save_file_path)
+    client().download_file(S3_BUCKET, s3_filepath, save_file_path)
```

Comment:
just a nit... thinking we should make s3.client() private, which will nudge folks to just use our s3.download_file